src/ansys/api/sherlock/v0/SherlockAnalysisService.proto

```diff
@@ -362,63 +363,93 @@ message UpdateICTAnalysisPropsRequest {
     optional string forceModelRebuild = 7;				// Model Creation. Valid values are "FORCE" or "AUTO".
     optional int32 ictResultCount = 8;					// Number of ICT layers to generate when using thermal analysis.
   }
-  
-  repeated ICTAnalysis ictAnalysisProperties = 2;
+
+  repeated ICTAnalysis ictAnalysisProperties = 2;		// List of ICTAnalysis properties to update for each CCA in a project.
+}
+
+/**
+ * Request to update the Parts List Validation properties.
+ */
+message UpdatePartsListValidationPropsRequest {
+  string project = 1;												// Sherlock Project name.
+
+  message PartsListValidation {
+    enum Matching {
```

Comment:
Seems to have the same purpose as MatchingMode in SherlockPartsService.proto.

Reply:
I'll remove this and refactor.

Reply:
Done